Source/Scene/ModelInstanceCollection.js

```diff
@@ -72,6 +72,7 @@ define([
      * @param {Boolean} [options.asynchronous=true] Determines if model WebGL resource creation will be spread out over several frames or block until completion once all glTF files are loaded.
      * @param {Boolean} [options.debugShowBoundingVolume=false] For debugging only. Draws the bounding sphere for the collection.
      * @param {Boolean} [options.debugWireframe=false] For debugging only. Draws the instances in wireframe.
+     * @param {ShadowMode} [options.shadows] Determines the shadowmode to be used for the Models.
```

Comment:
For organization purposes can you place this right after the `options.asynchronous` line. Also consider rewording the description to match `Model.js`, but change 'model' to 'collection'.

```javascript
* @param {ShadowMode} [options.shadows=ShadowMode.ENABLED] Determines whether the model casts or receives shadows from each light source.
```